libraries/ui-library/src/components/six-header/test/six-header.spec.tsx

```diff
@@ -67,7 +67,7 @@ describe('six-header', () => {
     `);
   });
 
-  it('should render app switcher', async () => {
+  xit('should render app switcher', async () => {
```

Comment:
Hm same here why is this test muted? I'm not a fan of bringing dead code into the main branch. Is the test currently broken?

Reply:
This test is currently broken, that is why I deactivated it for now. 

Do you want me to remove it instead? Or should we dedicate specific time to get those unit tests running. Then I would address this on a separate issue as the goal of this PR was to introduce the existing tests without modificaiton.

Reply:
were adressed